user-data.txt

```diff
@@ -48,6 +48,10 @@ cloud.aws:
   region: "${region}"
 EOF
 
+# Must be changed when bootstrap.mlockall: true
+sed -ie "s%^#LimitMEMLOCK=infinity$%LimitMEMLOCK=infinity%" /usr/lib/systemd/system/elasticsearch.service
```

Comment:
If you're changing a systemd unit file you need to reload it via `sudo systemctl daemon-reload`